### ISSUE TITLE
Close native apps instead of moving them to background

### ISSUE THREAD

**Issue by sshann** [S2-group/android-runner#45]
Currently, the Android Runner `close subject` does not terminate the app. Rather, it only goes back to the home screen and leaves the app "open" in the background. 

On my experiment, I had to actually finalise the app and open it again on each run. I tried to search on methods to automatically close the app but didn't found anything. My alternative to this issue was to add an interaction script in the event hook `after-run` to click in mobile device button that shows all open apps in the background and click to close the app from the last run. However, this approach is way too specific to a mobile device (since I used X and Y coordinates for the clicks). If an automatic and general method to resolve this is known, it would be nice to have it in AR, where the user can select in the JSON configuration file whether to leave the app in the background or to properly close it.

**Comment by EricZielinski** [comment]
Setting `clear_cache: true` in the configuration file will close the application window and make the runs cleaner.  The force-stop command does stop the application, but the application state is saved in memory and will be removed eventually if need be by the OS.

**Comment by sshann** [comment]
This was not an option in my experiment as I had to keep data between runs. The `clear_cache` configuration would clear the entire user data directory. Since I was testing a technique that learns the user behaviour after every session/run/repetition, I could not clear the entire user data directory. Furthermore, some of the android apps selected as subjects for my experiment requires the user to login before using the apps. Clearing the entire user data directory would automatically log out from the app.

So, instead of using the `clear_cache` configuration, I manually cleared out the unnecessary user data (cache and others) via a Python script in the AR event hooks. 

Not sure how often this scenario will occur for experiments designed using AR.
